docs/gettingstarted/modfiles.md

```diff
@@ -81,87 +48,90 @@ modId = "examplemod1"
+| `referralUrl`  | string  |   *nothing*   |                                     A URL to the download page of the dependency. Currently unused.                                     | `referralUrl="https://library.example.com/"` |
 
 :::danger
-The `ordering` of two mods may cause a crash due to a cyclic dependency: for example, mod A must load `"BEFORE"` mod B and mod B `"BEFORE"` mod A.
+The `ordering` of two mods may cause a crash due to a cyclic dependency, for example if mod A must load `"BEFORE"` mod B and at the same time, mod B must load `"BEFORE"` mod A.
 :::
 
 Mod Entrypoints
 ---------------
 
-Now that the `mods.toml` is filled out, we need to provide an entrypoint to being programming the mod. Entrypoints are essentially the starting point for executing the mod. The entrypoint itself is determined by the language loader used in the `mods.toml`.
+Now that the `mods.toml` is filled out, we need to provide an entrypoint for the mod. Entrypoints are essentially the starting point for executing the mod. The entrypoint itself is determined by the language loader used in the `mods.toml`.
 
 ### `javafml` and `@Mod`
 
-`javafml` is a language loader provided by Forge for the Java programming language. The entrypoint is defined using a public class with the `@Mod` annotation. The value of `@Mod` must contain one of the mod ids specified within the `mods.toml`. From there, all initialization logic (e.g., [registering events][events], [adding `DeferredRegister`s][registration]) can be specified within the constructor of the class. The mod bus can be obtained from `FMLJavaModLoadingContext`.
+`javafml` is a language loader provided by NeoForge for the Java programming language. The entrypoint is defined using a public class with the `@Mod` annotation. The value of `@Mod` must contain one of the mod ids specified within the `mods.toml`. From there, all initialization logic (e.g. [registering events][events] or [adding `DeferredRegister`s][registration]) can be specified within the constructor of the class.
 
 ```java
-@Mod("examplemod") // Must match mods.toml
+@Mod("examplemod") // Must match a mod id in the mods.toml
 public class Example {
-
   public Example() {
     // Initialize logic here
-    var modBus = FMLJavaModLoadingContext.get().getModEventBus();
-
-    // ...
   }
 }
 ```
 
+:::note
+There must be a 1-to-1 matching of mods in the `mods.toml` file and `@Mod` entrypoints. This means that for every mod defined, there must be exactly one `@Mod` annotation with that mod's id.
+:::
+
 ### `lowcodefml`
 
 `lowcodefml` is a language loader used as a way to distribute datapacks and resource packs as mods without the need of an in-code entrypoint. It is specified as `lowcodefml` rather than `nocodefml` for minor additions in the future that might require minimal coding.
```

Comment:
Please elaborate?

Reply:
I'm pretty sure for `lowcodefml` that the `@Mod` annotation is not needed on a class. To avoid confusion, I was suggesting to make an explicit reference to `javafml` saying that the `@Mod` annotation is unnecessary.